Widget/Shared Views/ArticleItemView.swift

```diff
@@ -65,6 +65,10 @@ struct ArticleItemView: View {
 		displayFormatter.dateStyle = .medium
 		displayFormatter.timeStyle = .none
 		
-		return displayFormatter.string(from: date!)
+		guard let dateFromDateFormatter = date else {
+			return ""
+		}
```

Comment:
How about we move the guard statement up a little, like this:

`guard let date = dateFormatter.date(from: dateString) …`

(This avoids the dateFromDateFormatter variable.)

Reply:
Reworked in 7df2fa5.